src/index.js

```diff
@@ -130,7 +130,15 @@ export default async function fetch(url, options_) {
 				const location = headers.get('Location');
 
 				// HTTP fetch step 5.3
-				const locationURL = location === null ? null : new URL(location, request.url);
+				let locationURL;
+				try {
+					locationURL = location === null ? null : new URL(location, request.url);
+				} catch (_error) {
```

Comment:
The catch binding is optional on all versions of Node.js that we support:

```suggestion
				} catch {
```